src/name_bazaar/server/contracts_api/registrar.cljs

```diff
@@ -17,19 +16,22 @@
                          :value (web3/to-wei 0.01 :ether)}
                         opts)))
 
-(defn transfer! [{:keys [:ens.record/hash :ens.record/label :ens.record/owner]} & [opts]]
-  (contract-call :name-bazaar-registrar
-                 :transfer
-                 (sha3 label hash)
-                 owner
-                 (merge {:gas 2000000}
-                        opts)))
-
-(defn entry [{:keys [:ens.record/hash :ens.record/label]}]
-  (parse-registrar-entry (contract-call :name-bazaar-registrar :entries (sha3 label hash))))
+(defn transfer! [{:keys [:ens.record/label :ens.record/owner]} & [opts]]
+  {:set-owner-tx
```

Comment:
Just making sure I understand this properly.

- Does this return a map with two keywords as a result of calling 2 smart contract calls?
- I do not see it being used anywhere, why just not call directly and return nothing? Am I missing something?

EDIT: I see it's used in later commit in tests.

Reply:
Right, we need transaction cost of both calls in tests. But I expect to use this in UI later, since it seems to be showing you all transaction you made.